crates/proof-of-sql/src/sql/proof_plans/filter_exec.rs

```diff
@@ -282,6 +298,7 @@ pub(super) fn prove_filter<'a, S: Scalar + 'a>(
     n: usize,
     m: usize,
 ) {
+    builder.push_one_evaluation_length(m);
```

Comment:
This is inside the `prove_filter`, but the corresponding `consume` is outside the `verify_filter`. Functionally it is correct, but we should keep the prove and verify mirrored. Either move them both in, or move them both out.